apps/judicial-system/backend/src/app/modules/case/interceptors/case.filter.ts

```diff
@@ -0,0 +1,8 @@
+import { CaseState, UserRole } from '@island.is/judicial-system/types'
+
+export function isStateVisibleToRole(
+  state: CaseState,
+  role: UserRole,
+): unknown {
```

Comment:
Afhverju skilar þetta unknown?

Reply:
That is unknown :)  I have no idea how this got there.